Date: 2023-08-09 12:31:49.74289541 +0200 CEST

    ● Test unhandled exception

      Crash!

        1 | test.fuzz("Test unhandled exception", jazzerBuffer => {
        2 | 	if (jazzerBuffer.toString() == "Fuzz") {
      > 3 | 		throw new Error("Crash!");
          | 		      ^
        4 | 	}
        5 | });
        6 |

        at unhandled-exception/UnhandledException.fuzz.js:3:9
        at result (node_modules/@jazzer.js/core/core.ts:357:15)

  Test Suites: 1 failed, 1 total
  Tests:       1 failed, 1 total
  Snapshots:   0 total
  Time:        0.425 s
  Ran all test suites matching /unhandled-exception/i.
  ==52093== ERROR: libFuzzer: fuzz target exited
  SUMMARY: libFuzzer: fuzz target exited
  MS: 4 ChangeByte-ChangeASCIIInt-CMP-EraseBytes- DE: "Fuzz"-; base unit: adc83b19e793491b1c6ea0fd8b46cd9f32e592fc
  0x46,0x75,0x7a,0x7a,
  Fuzz
  artifact_prefix='/tmp/cifuzz-nodejs-testdata-482095023/unhandled-exception/UnhandledException.fuzz/Test_unhandled_exception/'; Test unit written to .cifuzz-findings/poetic_fish/crashing-input
  Base64: RnV6eg==